app/views/govuk_component/breadcrumbs.raw.html.erb

```diff
@@ -14,7 +14,8 @@
             track_action: index + 1,
             track_label: crumb[:url],
             track_dimension: crumb[:title],
-            module: 'track-breadcrumb-click'
+            track_custom_dimension: 29,
```

Comment:
We could perhaps set this in a static variable with a name that explains when this is in case people wonder what the number 29 is all about. Something like: 

```
LINK_TEXT_CUSTOM_DIMENSION=29
```

And then use it throughout. What do you think?